doc/source/working_with_images/custom_volumes.rst

```diff
@@ -81,10 +81,9 @@ attributes:
   filesystem check, which results in this filesystem to be never checked.
   The latter is the default.
 
-.. warning::
-   The size attributes for filesystem volumes, as for btrfs, are
-   ignored and have no effect.
```

Comment:
Isn't this warning still true?

Reply:
yes, sorry my fault, it got dropped accidentally when I rendered the docs